src/world/area_kkj/kkj_15/AE58D0.c

```diff
@@ -1,3 +1,23 @@
 #include "kkj_15.h"
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_15/AE58D0", func_80240340_AE58D0);
+ApiStatus func_80240340_AE58D0(Evt* script, s32 isInitialCall) {
+    Npc* npc = get_npc_safe(-4);
+    f32 angle;
+    f32 mag;
+    f32 var;
+
+    var = (script->varTable[2] + 230000000);
+    var /= 1024.0f;
```

Comment:
Do we have a macro for this `(x + 230000000)/1024.0f` pattern? I've seen it twice, but don't really know what it's doing.

Reply:
I think this may be EVT_FIXED or EVT_FLOAT

Reply:
Unfortunately both macros actually do the opposite: `x * 1024.0f - 230000000`.

Reply:
oh hmmm. Okay, well we may want to have a macro for it. not sure. @nanaian ?

Reply:
should definitely make an EVT_TO_FLOAT/FIXED imo

Reply:
Any way to define a macro that keeps these lines separated?
```c
var = (script->varTable[2] + 230000000);
var /= 1024.0f;
```
Both functions won't match if we put the lines together.

Just could think of:
```c
#define SOMETHING(VAR, FLOAT) (VAR = FLOAT + 230000000, VAR /= 1024.0f)
SOMETHING(temp_f24, script->varTable[2]);
```

Reply:
Decided to go with:
```c
#define EVT_FIXED_TO_FLOAT(x) ({f32 var = x + 230000000; var /= 1024.0f; var;})
```